DevCycle/DVCVariable.swift

```diff
@@ -26,6 +26,7 @@ public class DVCVariable<T> {
         self.defaultValue = defaultValue
         self.isDefaulted = value == nil
         self.evalReason = evalReason
+        NotificationCenter.default.addObserver(self, selector: #selector(propertyChange(notification:)), name: NSNotification.Name.NewUserConfig, object: nil)
```

Comment:
maybe make this a private method so you dont have to do it the same way twice?

also is this going to cause performance issues with lots of variables?